This file is part of MXE. See LICENSE.md for licensing information.

From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Mark Brand <mabrand@mabrand.nl>
Date: Wed, 26 Mar 2014 23:45:05 +0100
Subject: [PATCH 1/1] workaround for stricmp decl problem on mingw32
 (mingw.org)


diff --git a/src/vmime/platforms/windows/windowsCodepages.hpp b/src/vmime/platforms/windows/windowsCodepages.hpp
index 1111111..2222222 100644
--- a/src/vmime/platforms/windows/windowsCodepages.hpp
+++ b/src/vmime/platforms/windows/windowsCodepages.hpp
@@ -32,6 +32,10 @@
 
 
 #include <string.h>
+#define stricmp _stricmp
+extern "C" {
+_CRTIMP int __cdecl __MINGW_NOTHROW     _stricmp (const char*, const char*);
+}
 
 
 namespace vmime {
